### PR TITLE
add try catch for logger, small updates to example app

### PR DESCRIPTION
**Pull request by cSparkle**
I was originally pulling the wrong type from the PluginCall for the setTag and breadcrumbs methods. I updated the typecasting to the appropriate types and made some small updates to the verbiage in the example app to make the messages clearer.

### CODE REVIEW

File: android/src/main/java/io/sentry/capacitor/SentryCapacitor.java [hunk 1]
```diff
@@ -326,21 +333,27 @@ public void clearBreadcrumbs() {
...
-        call.resolve();
+        if (call.getData().has("key") && call.getData().has("extra")) {
+            Sentry.configureScope(scope -> {
+                String key = call.getString("key");
+                String extra = call.getString("extra");
+                scope.setExtra(key, extra);
+            });
+
+            call.resolve();
+        }
```
Comment (joshdholtz) [inline android/src/main/java/io/sentry/capacitor/SentryCapacitor.java:344]: Does `call.resolve()` need to be executed outside of the if statement?

File: android/src/main/java/io/sentry/capacitor/SentryCapacitor.java [hunk 2]
```diff
@@ -326,21 +333,27 @@ public void clearBreadcrumbs() {
...
-        call.resolve();
+        if (call.getData().has("key") && call.getData().has("value")) {
+            Sentry.configureScope(scope -> {
+                String key = call.getString("key");
+                String value = call.getString("value");
+                scope.setTag(key, value);
+            });
+
+            call.resolve();
+        }
```
Comment (joshdholtz) [inline android/src/main/java/io/sentry/capacitor/SentryCapacitor.java:357]: Same here! Does this `call.resolve()` need to be executed outside of the if statement?

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Two small things! Otherwise 👍

File: android/src/main/java/io/sentry/capacitor/SentryCapacitor.java (hunk 1 above)
Comment (cSparkle) [reply android/src/main/java/io/sentry/capacitor/SentryCapacitor.java]: good catch!

**Pull request merged**